static/js/app.js

```diff
@@ -226,7 +234,7 @@ $(function() {
         },
         fetchMarkers : function() {
             if (!this.isReady) return;
-            this.updateUrl();
```

Comment:
Is the URL still getting updated when moving and zooming, after you removed this line?